step-maven-plugin/src/main/java/step/plugins/maven/LibraryConfiguration.java

```diff
@@ -0,0 +1,191 @@
+/*******************************************************************************
+ * Copyright (C) 2020, exense GmbH
```

Comment:
No need to fix, just as info: it seems that your license plugin doesn't use the current date